library/src/main/java/com/okta/oidc/OktaAuthenticationActivity.java

```diff
@@ -202,8 +203,19 @@ protected String getBrowser() {
     @VisibleForTesting
     protected Intent createBrowserIntent(String packageName, CustomTabsSession session) {
         CustomTabsIntent.Builder intentBuilder = new CustomTabsIntent.Builder(session);
-        if (mCustomTabColor > 0) {
-            intentBuilder.setToolbarColor(mCustomTabColor);
+        if (mCustomTabOptions != null) {
+            if (mCustomTabOptions.getCustomTabColor() != 0) {
+                intentBuilder.setToolbarColor(mCustomTabOptions.getCustomTabColor());
+            }
+            if (mCustomTabOptions.getStartExitResId() != 0) {
+                intentBuilder.setStartAnimations(this,
+                        mCustomTabOptions.getStartEnterResId(),
+                        mCustomTabOptions.getStartExitResId());
+            }
+            if (mCustomTabOptions.getEndEnterResId() != 0) {
```

Comment:
Why are we checking the exit res id for start but enter res id for end?

Reply:
If developers called the animation methods they should have set a correct id based on  @AnimRes. The check is to see if it's been called so one will suffice. If they set a value to 0 it is allowed but should be caught by AnimRes annotation.